src/components/Card.js

```diff
@@ -28,18 +42,37 @@ class Card {
     });
   }
 
-  _handleLikeButton = () => {
-    this._likeButton.classList.toggle("card__like-button_active");
-  };
-
-  _handleDeleteCard() {
+  handleDeleteCard() {
     this._cardElement.remove();
     this._cardElement = null;
   }
 
+  updateLikes(likes) {
+    const likeButton = this._cardElement.querySelector(".card__like-button");
+    if (this._isLiked) {
+      likeButton.classList.add("card__like-button_active");
+    } else {
+      likeButton.classList.remove("card__like-button_active");
```

Comment:
You have this code inside `renderLikes`. Please, call it here not to duplicate the code


```js

  updateLikes() {
    this.renderLikes();
    this._isLiked = !this._isLiked;
  }
```


But you have `updateLike` for that with the same code. Please, remove duplication of code